controlplane/pkg/model/model.go

```diff
@@ -103,7 +103,17 @@ func (i *impl) AddDataplane(dataplane *Dataplane) {
 	logrus.Infof("Dataplane added: %v", dataplane)
 }
 
-func (i *impl) DeleteDataplane(name string) {}
+func (i *impl) DeleteDataplane(name string) {
```

Comment:
What triggers the call to DeleteDataplane?

Reply:
Triggered from un register of data plane and if monitor find connection is down.

Reply:
Cool